src/breakpoints.jl

```diff
@@ -7,6 +7,37 @@ Return an array with all breakpoints.
 """
 breakpoints() = _breakpoints
 
+
+const breakpoint_update_hooks = []
+"""
+    on_breakpoints_updated(f)
+
+Register a one-argument function to be called after any update to the global set of
+breakpoints. This includes their creation, deletion, enabling and disabling.
+
+The function `f` should take two inputs:
```

Comment:
Might be clearer to say that this function will be called as `f(action, bp)`, where `action` is `breakpoint`, `enable`, `remove`, etc. (it might be important to spell out the entire list of actions that need to be handled, though), and `bp` is the breakpoint object. Then give an example of writing such an `f`, perhaps in the Documenter docs.